### PR TITLE
Added initial Environment Template to include a keyvault

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: Templates/enviroment.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      }
+    },
+    "microsoftAzureWebsitesRPObjectId": {
+      "type": "string",
+      "metadata": {
+        "description": "Object Id of the Azure Websites RP",
+        "environmentVariable": "microsoftAzureWebsitesRPObjectId"
+      }
+    },
+    "microsoftAzureFrontDoorObjectId": {
```
Comment (dalemorson) [inline Templates/enviroment.json]: Is the Front Door object Id required at the moment in the template?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+pr: none
+
+workspace:
+    clean: all
+
+variables:
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+pool:
```
Comment (ghost) [inline azure-pipelines.yml:17]: Change this to use one of our agent pools?

Comment (mwevill) [reply azure-pipelines.yml:17]: No these don't have access to CIP

**Review by ghost — APPROVED** [review]
Approved

**Pull request merged**